misc/testerina/modules/testerina-core/src/main/java/org/ballerinalang/testerina/core/BTestRunner.java

```diff
@@ -206,7 +208,8 @@ private void execute(boolean buildWithTests) {
             if (buildWithTests) {
                 return;
             }
-            outStream.println("No test functions found in the provided ballerina files.");
+            outStream.println("Running tests");
```

Comment:
I think this line should be common for both scenarios (tests found or not test found). So we can move this to a common place rather than having it separately for  both cases.

Reply:
Fixed with 5fc4cc0